ActindoWorkFlow/UserInterface/02a_Workflows.md

```diff
@@ -0,0 +1,253 @@
+
+- ![Undo](/Assets/Icons/Undo02.png "[Undo]") (Undo)  
+  Click this button to undo any changes made.
+
+- ![Redo](/Assets/Icons/Redo.png "[Redo]") (Redo)  
+  Click this button to redo any undone changes.
+
+- ![Comment](/Assets/Icons/Comment.png "[Comment]") (Comment)
+
+[comment]: <> (OLI: "Comment" button? Was macht es?)
+
+- [NEW ACTION]  
+  Click this button to add a new action. A window to search for an action is displayed, see [Search for an action](#search-for-an-action). For detailed information about creating an action, see [Create an action](ActindoWorkFlow/Operation/01_ManageWorkflows.md#create-an-action).
+
+
+The workflow diagram is displayed in the workflow editor. For detailed information about the workflow elements, see [Workflow elements](01_WorkflowElements.md).
+
+
+
+## Create view
+
+*Workflows > Workflows > Tab OVERVIEW > Button Points > Menu entry create*
+
+![Create view](/Assets/Screenshots/ActindoWorkFlow/Workflows/CreateView.png "[Create view]")
+
+
+- *Name*  
+Enter a name for the view.
+
+- [CANCEL]  
+Click this button to cancel creating a view. The *Create view* window is closed.
+
+- [SAVE]  
+Click this button to save the new view. The *Create view* window is closed. The view is saved and displayed in the *View* drop-down list.
+
+
+
+## New Workflow
+
+*Workflows > Workflows > Tab OVERVIEW > Button Add*
+
+![New workflow](/Assets/Screenshots/ActindoWorkFlow/Workflows/NewWorkflow.png "[New workflow]")
+
+
+- *Select a name for your new workflow*  
+Click this field to enter a name for the new workflow.
+
+- *Select a unique key for your new workflow*  
+Click this field to enter a key for the workflow.
+
+- *Choose the data type of your start place*  
+Click this field (search bar) to enter the name of the start place or a keyword to search for.
+
+  > [Info] The list of places is filtered for your keyword as you type.
+
+- ![Delete](/Assets/Icons/Cross02.png "[Delete]") (Delete)  
+  Click this button to delete the keyword and clear the search bar.
+
+- *Choose the data type of your end place*  
+Click this field (search bar) to enter the name of the end place or a keyword to search for.
+
+  > [Info] The list of places is filtered for your keyword as you type.
+
+- ![Delete](/Assets/Icons/Cross02.png "[Delete]") (Delete)  
+    Click this button to delete the keyword and clear the search bar.
+
+- [CANCEL]  
+Click this button to cancel creating a new workflow. The *New Workflow* window is closed.
+
+- [CREATE]  
+Click this button to create a new workflow. The workflow editor with the defined start and end places is displayed.
+
+
+
+## Search for an action
+
+*Workflows > Workflows > Tab OVERVIEW > Select workflow > Select version > Button NEW ACTION*
+
+![Search for an action](/Assets/Screenshots/ActindoWorkFlow/Workflows/SearchAction.png "[Search for an action]")
+
+The *Search for an action* window displays a list of all compatible actions for the selected place.
```

Comment:
Die Info ist nur korrekt, wenn ich über einen + Button an einem place in dieses Fenster gelange, nicht aber, wenn ich den Button NEW ACTION klicke. 
Die Info zu den zwei Möglichkeiten, wie ich in dieses Fenster gelange und dass einmal alle und einmal nur die zum place kompatiblen Actions angezeigt werden (vgl. Info in dem Fenster unter search bar) sollte hier angeführt werden